lib/ui/painting.dart

```diff
@@ -1217,20 +1249,20 @@ class Image extends NativeFieldWrapperClass2 {
 
   /// Converts the [Image] object into a byte array.
   ///
-  /// The image bytes will be RGBA form, 8 bits per channel, row-primary.
+  /// [format] specifies the format in which the bytes will be returned.
```

Comment:
avoid starting with a lowercase letter by saying something like "The [...] argument ..."